include/secp256k1_musig.h

```diff
@@ -149,7 +149,7 @@ SECP256K1_API int secp256k1_musig_pubkey_combine(
  *              msg32: the 32-byte message to be signed. Shouldn't be NULL unless you
  *                     require sharing public nonces before the message is known
  *                     because it reduces nonce misuse resistance. If NULL, must be
- *                     set with `musig_session_set_msg` before signing and verifying.
+ *                     set with `musig_session_get_public_nonce`.
```

Comment:
We should probably cut this comment down even further to get rid of the "unless you require" clause.

Reply:
Hm, not sure what exactly you're suggesting. Something like that?
```
 *              msg32: the 32-byte message to be signed. Shouldn't be NULL. If NULL, must be
 *                           set with `musig_session_get_public_nonce`.
```
Seems to be less clear. Also I assume sharing nonce commitments won't be a very exotic thing. Lightning will likely use it to avoid roundtrips. I have an open PR in the scriptless scripts repo (https://github.com/apoelstra/scriptless-scripts/pull/6) that explains how to do this because I was contacted by someone concerned that MuSig will end up being very expensive in communication costs.

Reply:
I think if you change "require sharing public nonces" to "require sharing nonce commitments" I'll be happy

Reply:
fixed